### DISCUSSION TITLE
unstable_metaTags only support <meta property=...> and not <meta name=...>

### DISCUSSION THREAD

**Discussion by richtera** [wevm/frog-archived#497] [Ideas]
It would be best if the unstable_metaTags objects supported both property and/or name.
Some of the og data expects name=

**Comment by dalechyn** [comment]
Do you think adding `name` as an allowed property would solve that?

**Reply by richtera** [reply]
Adding the t as an optional property and the rendering name= or property= or both would solve it for sure
